<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Sample Test MOBF"
           tests="13"
           failures="12"
           errors="0"
           skipped="0"
           time="0.009">
  <properties />
            <testcase name="QTYvalidation" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.QTYvalidation(SamplePO.java:160)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="GetOrder" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  Test.SamplePO.GetOrder(SamplePO.java:67)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="DATEvalidation" time="0.000" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.DATEvalidation(SamplePO.java:223)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="NARROWBY" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.NARROWBY(SamplePO.java:313)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="ViewOrder" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  Test.SamplePO.ViewOrder(SamplePO.java:92)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="Navigation" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.Navigation(SamplePO.java:266)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="ORDERvalidation" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.ORDERvalidation(SamplePO.java:129)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="PRICEvalidation" time="0.000" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.PRICEvalidation(SamplePO.java:193)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="SORTBY" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.SORTBY(SamplePO.java:300)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="OrderList" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.OrderList(SamplePO.java:253)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="UrgentOrder" time="0.001" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  Test.SamplePO.UrgentOrder(SamplePO.java:115)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
            <testcase name="SWIPE" time="0.000" classname="Test.MyOrder">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  Test.SamplePO.SWIPE(SamplePO.java:372)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="T1" time="0.000" classname="Test.MyOrder" />
      
</testsuite>
